scala/dllib/src/main/scala/com/intel/analytics/bigdl/dllib/example/nnframes/xgboost/README.md

```diff
@@ -0,0 +1,64 @@
+# XGBoostClassifier Train Example
+
+## Environment
+- spark 2.4.6
+- BigDL 2.0 
+## Data Prepare
+
+### Build BigDL/scala
```

Comment:
Maybe we can show how to download the prebuilt jar package as we recommend users to use the prebuild jar package

Reply:
I have added download url in README.md